bark/src/bin/bark/main.rs

```diff
@@ -202,16 +211,32 @@ async fn inner_main(cli: Cli) -> anyhow::Result<()> {
 		Command::GetAddress => println!("{}", w.get_new_onchain_address()?),
 		Command::GetVtxoPubkey => println!("{}", w.vtxo_pubkey()),
 		Command::Balance => {
-			info!("Onchain balance: {}", w.onchain_balance().await?);
-			info!("Offchain balance: {}", w.offchain_balance().await?);
-			let (claimable, unclaimable) = w.unclaimed_exits().await?;
-			if !claimable.is_empty() {
-				let sum = claimable.iter().map(|i| i.spec.amount).sum::<Amount>();
-				info!("Got {} claimable exits with total value of {}", claimable.len(), sum);
-			}
-			if !unclaimable.is_empty() {
-				let sum = unclaimable.iter().map(|i| i.spec.amount).sum::<Amount>();
-				info!("Got {} unclaimable exits with total value of {}", unclaimable.len(), sum);
+			let onchain = w.onchain_balance().await?;
+			let offchain =  w.offchain_balance().await?;
+			let (claimables, unclaimables) = w.unclaimed_exits().await?;
+			let claimable = claimables.iter().map(|i| i.spec.amount).sum::<Amount>();
+			let unclaimable = unclaimables.iter().map(|i| i.spec.amount).sum::<Amount>();
+			if cli.json {
+				//TODO(stevenroose) should make a bark-json crate with these structs
+				serde_json::to_writer(io::stdout(), &json!({
+					"onchain": onchain.to_sat(),
+					"offchain": offchain.to_sat(),
+					"claimable": claimable.to_sat(),
+					"unclaimable": unclaimable.to_sat(),
+				})).unwrap();
```

Comment:
The chosen names are pretty confusing here. I would expect that the meaning of each field is obvious in a user-facing data-structure.

It took me a few minutes and a deep-dive in the source to understand what `claimable` and `unclaimable` means.

As a user it might be quite concerning to see you have a fairly large `offchain` balance and that none of it is `claimable`.

I understand that `claimable` and `unclaimable` only refers to transaction that took/are taking part in a unilateral exit.

Can we rename `unclaimable` to `exit_pending`?

For the `claimable` funds I see two options.
- List it under `onchain`-balance
- List it under `exit_awaiting-claim`

Reply:
Hmm, good point. I copied the variable names without much thinking.

I would say `exit_pending` and `exit_claimable`? Tbh, we should probably have some kind of mode where the wallet automatically claims the outputs if it can.

Adding to onchain is wrong as the ASP can still take the money in theory if the user doesn't claim before the expiry.

Reply:
I fully agree with your comment.

I would approve every option you describe

Reply:
Renamed with the suggestion I made.